test/endtoend/vtorc_vtadmin_test.sh

```diff
@@ -111,6 +112,19 @@ function verifyVtadminSetup() {
   chromiumHeadlessRequest "http://localhost:14000/keyspace/example/commerce/shards" "commerce/-"
 }
 
+# verifyVTOrcSetup verifies that VTOrc is running and repairing things that we mess up
+function verifyVTOrcSetup() {
+  # Stop replication using the vtctld and wait for VTOrc to repair
+  allReplicaTablets=$(getAllReplicaTablets)
+  for replica in $(echo "$allReplicaTablets") ; do
+    vtctlclient StopReplication "$replica"
+  done
+  # Now that we have stopped replication on both the tablets, we know that this will
+  # only succeed if VTOrc is able to fix it since we are running vttablet with disable active reparent
+  # and semi-sync durability policy
+  mysql -e "insert into customer(email) values('newemail@domain.com');"
```

Comment:
I am not sure how fast `StopReplication` is, but if it is not, maybe we should have some delay before calling the `mysql` command so we can ensure that the replication is stopped in the replicas.

Reply:
The command won't return until replication is stopped. We don't need any wait after it.

Reply:
Okay that's perfect!